builtin/credential/aws/backend.go

```diff
@@ -17,7 +17,10 @@ import (
 	cache "github.com/patrickmn/go-cache"
 )
 
-const amzHeaderPrefix = "X-Amz-"
+const (
+	amzHeaderPrefix        = "X-Amz-"
+	operationPrefixAWSAuth = "aws-auth"
```

Comment:
Would it make sense to have the naming go from general to specific on the ordering, and swap it to be `auth-aws`?

Reply:
Swapped the "auth" to the suffix as we discussed in slack.